### ISSUE TITLE
gravatar images are blurry

### ISSUE THREAD

**Issue by peterbsmyth** [openhacksyr/openhacksyr.github.io#10]

**Comment by vormwald** [comment]
fix == `[gravatar_url]?size=160`?

**Comment by peterbsmyth** [comment]
Yep and into the members.html file so not every member entry needs that
query. I'm gonna do it later today.

On Friday, October 30, 2015, Mike Vormwald notifications@github.com wrote:

> fix == [gravatar_url]?size=160?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/openhacksyr/openhacksyr.github.io/issues/10#issuecomment-152567430
> .

## 

Sent from Gmail Mobile

**Comment by vormwald** [comment]
Thanks @peterbsmith2 ! If you're in there poking around, you mind checking: https://github.com/openhacksyr/openhacksyr.github.io/issues/11 ?
